ui/widgets/controlgroup.js

```diff
@@ -112,6 +112,11 @@ return $.widget( "ui.controlgroup", {
 				.each( function() {
 					var element = $( this );
 					var instance = element[ widget ]( "instance" );
+
+					// If the button is the child of a spinner ignore it
+					if ( widget === "button" && element.parent( ".ui-spinner" ).length ) {
```

Comment:
I agree I just was not sure how else to do this unless we force enhancment within a controlgroup to be more explicit ala JQM data-role